### PR TITLE
Footer import error fixed

### PR DESCRIPTION
**Pull request by sayinmehmet47**

### CODE REVIEW

**Comment by JoshuaKGoldberg** [comment]
Hey @sayinmehmet47, what error are you trying to fix here? The code seems to run fine for me, and I prefer named exports over defaults...?

**Comment by sayinmehmet47** [comment]
@JoshuaKGoldberg I cloned the repo, and when I do 

``` js
npm install
```


the project can not start because of the some errrors I fixed. In your youtube tutorial it start directly after install dependencies. But in my machine it fired these small errors. I wanted to fix them.

**Comment by JoshuaKGoldberg** [comment]
Ah, got, thanks! The default settings somewhere must have changed - this used to work out of the box 😄.

The issue isn't with using a default export or not; it's with case sensitivity. https://github.com/JoshuaKGoldberg/next-js-to-ts/blob/2a03187fa98abca404468ead7c8519eabb01d884/src/components/Layout.js#L3

The file name is `Footer.js`, not `footer.js`.

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
The only change you should need to make is in `src/components/Layout.js`, switching the lower-case `f` in `"./footer;"` to `F`.

Thanks for sending this in!

**Comment by sayinmehmet47** [comment]
@JoshuaKGoldberg  anyway thank you so much also for tutorial:) It helps me a lot to implement typescript in my project

File: src/components/Footer.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import Image from "next/image";
 
 import styles from "./Footer.module.css";
 
-export function Footer() {
+export default function Footer() {
```
Comment (JoshuaKGoldberg) [inline src/components/Footer.js]: There's no need to change this.

```suggestion
export function Footer() {
```

File: src/components/Footer.test.js [hunk 2]
```diff
@@ -1,15 +0,0 @@
...
-  children: "Hello, world!",
-  description: "my description",
-  title: "my title",
-});
-
-describe(Footer, () => {
-  it("renders", () => {
-    renderView();
-  });
-});
```
Comment (JoshuaKGoldberg) [inline src/components/Footer.test.js]: 😢 but I like these tests!

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]

File: src/components/Footer.test.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { setupRtl } from "component-test-setup";
+import Footer from "./Footer";
 
-import { Footer } from "./Footer";
```
Comment (JoshuaKGoldberg) [inline src/components/Footer.test.js]: There shouldn't be any changes to this file. The import here was correct before.

* `import Footer from "./Footer";` is a _default_ import. It retrieves the object under the name `default` from `"./Footer"`, and stores it in a variable named `Footer`.
* `import { Footer } from "./Footer";` is a _named_ import. It retrieves the object under the name `Footer` from `"./Footer"`, and stores it in a variable named `Footer`.

https://stackoverflow.com/questions/46913851/why-and-when-to-use-default-export-over-named-exports-in-es6-modules

File: src/components/Layout.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import Head from "next/head";
+import Footer from "./Footer";
+
 
-import { Footer } from "./footer";
```
Comment (JoshuaKGoldberg) [inline src/components/Layout.js:3]: Same here: the issue was casing, not switching from named to default.

File: src/components/Layout.js [hunk 5]
```diff
@@ -13,8 +14,8 @@ export function Layout({ children, description, title }) {
       </Head>
 
       <main className={styles.main}>{children}</main>
-
-      <Footer />
+    <Footer/>
```
Comment (JoshuaKGoldberg) [inline src/components/Layout.js]: Please revert this line's change. It was nicely formatted before. 🙂

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
Closer!

File: src/components/Footer.test.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import { setupRtl } from "component-test-setup";
+import {Footer} from "./Footer";
```
Comment (JoshuaKGoldberg) [inline src/components/Footer.test.js]:
```suggestion

import { Footer } from "./Footer";
```

**Review by JoshuaKGoldberg — APPROVED** [review]
Thanks for this fix @sayinmehmet47! And sorry for taking so long to merge - I was waiting for one last fix to another file, but it was small and I just did it myself.

We should note that the workshop was structured to intentionally show that this file casing is something caught by TypeScript. Since it seems to not work out of the box on Linux/Mac, we'll just have to go without that finding. Ah well. 🙂

**Comment by sayinmehmet47** [comment]
@JoshuaKGoldberg  Thanks for merging it , it takes long time but thanks for merging it

**Pull request merged**